DAO/UserDAO.py

```diff
@@ -0,0 +1,23 @@
+
+
+import psycopg2
+
+from config.config import BaseConfig
+
+
+class UserDAO:
+
+    def __init__(self):
+        config = BaseConfig()
+
+        self.conn = psycopg2.connect(dbname=config.DBNAME, user=config.USER, password=config.PASSWORD, host=config.HOST,
+                                     port=config.PORT)
+
+    def insert_user(self, username, password, first_name, last_name, email, phone_number):
+        cursor = self.conn.cursor()
+        query = "insert into users (username, password, first_name, last_name, email, phone_number) " \
```

Comment:
To avoid injections, use execute with %s in place of each variable, then pass the value via a list or tuple as the second parameter of execute. [Stackoverflow](https://stackoverflow.com/questions/7929364/python-best-practice-and-securest-to-connect-to-mysql-and-execute-queries)

```python
query = 'insert into users (username, password, first_name, last_name, email, phone_number) values (%s,%s,%s,%s,%s,%s) returning uid'
```

Reply:
That's what I did

Reply:
oh I see it now